lib/codegen/AWS_LAMBDA-request-mapping-template.tmpl.ejs

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "2017-02-28",
+  "operation": "Invoke",
+  "payload": {
+      "field": "<%= field %>",
+      "arguments":  $utils.toJson($context.arguments)
```

Comment:
@merunga What about `source` and `ìdentity` here? 
I would simply do this: 

"payload": {
	"field": "<%= field %>",
    "context":  $utils.toJson($context)
}

You don't know what the future is made of. AWS might add new fields in `context` in the future.